modules/openapi-generator/src/main/resources/cpp-qt5-client/CMakeLists.txt.mustache

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.2)
+
+project(client)
+set(CMAKE_VERBOSE_MAKEFILE ON)
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+set(CMAKE_AUTOMOC ON)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -std=c++14 -Wall -Wno-unused-variable")
+
+find_package(Qt5Core REQUIRED)
+find_package(Qt5Network REQUIRED)
+
+file(GLOB SRCS
+    ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp
+)
+
+include_directories(
+    ${Qt5Core_INCLUDE_DIRS}
+    ${Qt5Network_INCLUDE_DIRS}
+)
+
+add_library(${PROJECT_NAME} ${SRCS})
+target_link_libraries(${PROJECT_NAME} PRIVATE Qt5Core Qt5Network ssl crypto)
+
+set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 14)
+set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD_REQUIRED ON)
+set_property(TARGET ${PROJECT_NAME} PROPERTY CMAKE_CXX_EXTENSIONS OFF)
```

Comment:
Sorry that I confused you :grin:

it should be `CXX_EXTENSIONS`, would be `CMAKE_CXX_EXTENSIONS` if we'd use it in global context.

Reply:
done